HousingSearchListener/V1/Infrastructure/Exceptions/PersonNotFoundException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace HousingSearchListener.V1.Infrastructure.Exceptions
+{
+    public class PersonNotFoundException : EntityNotFoundException
```

Comment:
I am thinking we could just call this an EntityNotFoundException and make it generic since the functionality seems similar?